gsplat/cuda/csrc/bindings.h

```diff
@@ -121,6 +121,18 @@ std::tuple<
     const torch::Tensor &background
 );
 
+std::tuple<torch::Tensor, torch::Tensor, torch::Tensor>
+rasterize_indices_tensor(
+    const std::tuple<int, int, int> tile_bounds,
+    const std::tuple<int, int, int> block,
```

Comment:
can we update this interface so that we hide the block size from the user as in #129 ?